www/src/components/Menu.tsx

```diff
@@ -144,16 +149,43 @@ interface IManuQueryComponents {
   all: Array<IComponentNavItem>
 }
 
+const foundationLinks = [
+  'Colors', 'Elevation', 'Typography', 'Layout', 'Spacing', 'Icons', 'CSS-Utilities', 'Responsive',
+];
+
 function Menu() {
+  const {
+    settings,
+    handleSettingsChange,
+  } = useContext(SettingsContext);
   const { components } = useStaticQuery(menuQuery);
-  const { categories, all }: IManuQueryComponents = components;
+  const { categories }: IManuQueryComponents = components;
```

Comment:
[clarification] Is this supposed to say "Manu" or "Menu"?

Reply:
Yes, thanks
Good catch